app/models/framework/definition/RM858.fdl

```diff
@@ -13,7 +13,7 @@ Framework RM858 {
     optional CustomerPostCode from 'Customer PostCode'
     CustomerName from 'Customer Organisation'
     CustomerURN from 'Customer URN'
-    optional InvoiceDate from 'Customer Invoice Date'
+    InvoiceDate from 'Customer Invoice Date'
```

Comment:
This appears to be doing the opposite of what the commit message says.

Reply:
You're right – the commit message is incorrect. Changing